server/passport/signup-strategy.js

```diff
@@ -0,0 +1,13 @@
+import { Strategy } from 'passport-local';
+import UserFacade from '../user/user-facade';
+
+export default new Strategy(async (username, password, done) => {
```

Comment:
Usually there is no need to create signup strategy. You just create `/signup` route and process new user with UserController.